app/assets/stylesheets/modules/_landing.scss

```diff
@@ -0,0 +1,41 @@
+.landing {
+  .banner {
```

Comment:
No problem, I'll add `.landing` to the inner classes too, to be consistent with the sponsors page!